### ISSUE TITLE
Setting HADOLINT_TRUSTED_REGISTRIES has no effect

### ISSUE THREAD

**Issue by evhan** [hadolint/hadolint#756]
- [x] This is a bug report
- [ ] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior

Running `HADOLINT_TRUSTED_REGISTRIES="something" hadolint Dockerfile` is equivalent to `hadolint --trusted-registry something Dockerfile`.

### Actual behavior

Setting `HADOLINT_TRUSTED_REGISTRIES` has no effect.

### Steps to reproduce the behavior

The README mentions that this environment variable can be used in lieu of `--trusted-registry`:

> In addition to config files, Hadolint can be configured with environment variables.
> 
> ```
> HADOLINT_TRUSTED_REGISTRIES              # comma separated list of registry urls
> ```

I glanced at `Environment.hs` and it looks like it's [checking for `HADOLINT_ALLOWED_REGISTRIES`](https://github.com/hadolint/hadolint/blob/d16f342c8e70fcffc7a788d122a1ba602075250d/src/Hadolint/Config/Environment.hs#L30) instead.

I'm happy to open a PR for this, just not sure in which direction it should be fixed. Probably the environment variable and README should be updated?

Hadolint version:

```bash
Haskell Dockerfile Linter v2.8.0-0-g398770f-dirty
```

**Comment by m-ildefons** [comment]
Hi @evhan ,
thanks for the bug report. Indeed, I messed this up in https://github.com/hadolint/hadolint/commit/bf7e48ea735db423a349fb2d7a83fe7a77091cc2
I think it would be best to have consistency with the other configuration mechanisms and change the environment variable to `HADOLINT_TRUSTED_REGISTRIES`, as mentioned in the readme.
If you would open a PR for that, I'd be greatful or I'll do it if you don't find the time.

**Comment by evhan** [comment]
> If you would open a PR for that, I'd be greatful or I'll do it if you don't find the time.

All good, done.

**Comment by Stazz-Sphinx** [comment]
Confirm that issue exists with ignore trusted repositories listed in `trustedRegistries` section in .hadolint.yaml config file.  The key `--trusted-registry` works fell from cli. 

OS: macos Monterey 12.1 
hadolit version  - doesn't show via hadolint -v|--version but brew show that version is 

```
✗ brew info hadolint
hadolint: stable 2.8.0 (bottled)
```
